src/arviz_plots/plots/compareplot.py

```diff
@@ -0,0 +1,137 @@
+"""Compare plot code."""
+from importlib import import_module
+
+from arviz_base import rcParams
+
+
+def plot_compare(
+    cmp_df,
+    color="black",
+    similar_band=True,
+    relative_scale=False,
+    figsize=None,
+    target=None,
+    backend=None,
+):
+    r"""Summary plot for model comparison.
+
+    Models are compared based on their expected log pointwise predictive density (ELPD).
+
+    Notes
+    -----
+    The ELPD is estimated either by Pareto smoothed importance sampling leave-one-out
+    cross-validation (LOO) or using the widely applicable information criterion (WAIC).
+    We recommend LOO in line with the work presented by [1]_.
+
+    Parameters
+    ----------
+    comp_df : pandas.DataFrame
+        Result of the :func:`arviz.compare` method.
+    color : str, optional
+        Color for the plot elements. Defaults to "black".
+    similar_band : bool, optional
+        If True, a band is drawn to indicate models with similar
+        predictive performance to the best model. Defaults to True.
+    relative_scale : bool, optional.
+        If True scale the ELPD values relative to the best model.
+        Defaults to False.
+    figsize : (float, float), optional
+        If `None`, size is (10, num of models) inches.
+    target : bokeh figure, matplotlib axes, or plotly figure optional
+    backend : {"bokeh", "matplotlib", "plotly"}
+        Select plotting backend. Defaults to rcParams["plot.backend"].
+
+    Returns
+    -------
+    axes :bokeh figure, matplotlib axes or plotly figure
+
+    See Also
+    --------
+    plot_elpd : Plot pointwise elpd differences between two or more models.
+    compare : Compare models based on PSIS-LOO loo or WAIC waic cross-validation.
+    loo : Compute Pareto-smoothed importance sampling leave-one-out cross-validation (PSIS-LOO-CV).
+    waic : Compute the widely applicable information criterion.
+
+    References
+    ----------
+    .. [1] Vehtari et al. (2016). Practical Bayesian model evaluation using leave-one-out
+       cross-validation and WAIC https://arxiv.org/abs/1507.04544
+    """
+    information_criterion = ["elpd_loo", "elpd_waic"]
+    column_index = [c.lower() for c in cmp_df.columns]
+    for i_c in information_criterion:
+        if i_c in column_index:
+            break
+    else:
+        raise ValueError(
+            "cmp_df must contain one of the following "
+            f"information criterion: {information_criterion}"
+        )
+
+    if backend is None:
+        backend = rcParams["plot.backend"]
+
+    if relative_scale:
+        cmp_df = cmp_df.copy()
+        cmp_df[i_c] = cmp_df[i_c] - cmp_df[i_c].iloc[0]
+
+    if figsize is None:
+        figsize = (10, len(cmp_df))
```

Comment:
This should be set in dots, matplotlib can handle dots without much issue whereas the rest can't handle inches at all. figsize related functions take both `figsize` and `figsize_units` so both work (default is inches for matplotlib, dots for the rest), but things we want to be proper backend agnostic should use dots insead. Multiplying by 100 or so should make the trick